redwood-lazylayout-compose/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/LazyListState.kt

```diff
@@ -18,32 +18,71 @@ package app.cash.redwood.lazylayout.compose
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.Saver
+import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
 
 @Composable
-public fun rememberLazyListState(
-  initialFirstVisibleItemIndex: Int = 0,
```

Comment:
Why did we remove the default value here? It's kinda nice to have it as it aligns with what Compose UI does with their `LazyListState` implementation.

Reply:
the init for `initialFirstVisibleItemIndex` now is at down below (to 0)